src/runtime_src/core/common/query_requests.h

```diff
@@ -1073,6 +1075,58 @@ struct error : request
   }
 };
 
+// Retrieve asynchronous xocl errors from xocl driver
+struct xocl_errors : request
+{
+  using result_type = std::vector<char>;
+  static const key_type key = key_type::xocl_errors;
```

Comment:
I mean could this be key_type::error ?  I think the idea is that both zocl and xocl will respond to the same query request and return the same format, if so then when that happens we can simply remove existing `struct error` and rename this struct to error?

Reply:
Yes it will be changed to same name for zocl & xocl. maybe "errors"